container/tabs.go

```diff
@@ -29,22 +29,22 @@ func (ti *TabItem) enable() {
 	}
 }
 
+func (ti *TabItem) disable() {
```

Comment:
private functions (not exported) should be below public (exported) ones.
Apologies that this is not automatically enforced yet, we are working on it.
More info about order of code in a file at: https://github.com/fyne-io/fyne/wiki/Contributing#code-style

Reply:
Sorry about that! I admit I should have read through that a bit more thoroughly the first time. I believe things are correct now.